config.js

```diff
@@ -4,11 +4,18 @@
 //
 //
 var trace = true; // trace entry and exit for debugging
-
-
+var generate_item_list = true; //generate the item, weapon, and armor lists for the inventory tab on startup
```

Comment:
do we ever *not* want to do this?

Reply:
It only really needs to be done when the prices list changes or you might want to use hand edited lists.